document.ts

```diff
@@ -1,11 +1,28 @@
 import { CortexApiClient } from "./api-client.js";
 import { Catalog } from "./catalog.js";
 
+export type UrlContentType = "url";
```

Comment:
Would it be better for discoverability / intellisense experience to use ContentTypeXYZ naming pattern?

Reply:
In hindsight yes. Probably a change we should make at some point, but for now I'm just pattern matching against the existing content types.